src/app/components/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { IoMdMail } from "react-icons/io";
+import { FaLinkedin } from "react-icons/fa";
+
+const year = new Date().getFullYear();
+
+const Footer: React.FC = () => {
+  return (
+    <footer style={{
+        width: '100%',
```

Comment:
Can you use the tailwind equivalent classes instead of hardcoding it in the html here? Let me know if you need help with this.